oap-logstream/src/main/java/oap/logstream/LogStreamProtocol.java

```diff
@@ -3,8 +3,21 @@
 import oap.message.MessageProtocol;
 
 
-public final class LogStreamProtocol {
-    public static final byte MESSAGE_TYPE = 20;
-    public static final short STATUS_BACKEND_LOGGER_NOT_AVAILABLE = 20000;
-    public static final short STATUS_OK = MessageProtocol.STATUS_OK;
+@SuppressWarnings( "checkstyle:InterfaceIsType" )
+public interface LogStreamProtocol {
+    ProtocolVersion CURRENT_PROTOCOL_VERSION = ProtocolVersion.BINARY_V2;
+
+    enum ProtocolVersion {
+        OLD_TSV( 1 ), BINARY_V2( 2 );
```

Comment:
```suggestion
        TSV_ONLY( 1 ), BINARY_V2( 2 );
```
Maybe `OLD_TSV` -> `TSV_ONLY`? The tsv format itself is not old, but `1` protocol version supported only TSV (without any other binary or non-binary format)